### PR TITLE
ocrvs-5257 configurable child identifiers

### PR DESCRIPTION
**Pull request by Zangetsu101**
Farajaland PR: https://github.com/opencrvs/opencrvs-farajaland/pull/575

### CODE REVIEW

**Comment by Zangetsu101** [comment]
needs amends because of #5491

File: packages/client/src/forms/configuration/default/birth.ts [hunk 1]
```diff
@@ -989,7 +989,46 @@ export const birthRegisterForms: ISerializedForm = {
...
+          operation: 'childIdentityToFieldTransformer',
+          parameters: [
+            [
+              IdentityIdType.BirthConfigurableIdentifier_1,
+              IdentityIdType.BirthConfigurableIdentifier_2,
+              IdentityIdType.BirthConfigurableIdentifier_3
+            ]
+          ]
+        }
+      }
```
Comment (euanmillar) [inline packages/client/src/forms/configuration/default/birth.ts:1031]: @Zangetsu101 now that these are in Farajaland, can you please move them to the birth form there?  https://github.com/opencrvs/opencrvs-farajaland/pull/575

File: packages/client/src/utils/gateway.ts [hunk 2]
```diff
@@ -1008,6 +1008,9 @@ export type Identifier = {
 
 export enum IdentityIdType {
```
Comment (euanmillar) [inline packages/client/src/utils/gateway.ts:1009]: @Zangetsu101 I dont think these are required anymore as any id type can be sent now?

**Comment by euanmillar** [comment]
@Zangetsu101 I changed the base branch so that you can move the mapping functions into the correct location in the new form config location in https://github.com/opencrvs/opencrvs-farajaland/pull/575

File: packages/client/src/utils/gateway.ts (hunk 2 above)
Comment (Zangetsu101) [reply packages/client/src/utils/gateway.ts]: yeah I've made them normal strings now in the mapping functions

**Pull request merged**